app/jobs/spotlight/process_bulk_updates_csv_job.rb

```diff
@@ -13,15 +13,25 @@ def perform(exhibit, bulk_update)
       errors = 0
       header_converter = ->(header) { header } # Use raw header for columns (since they are configured)
       csv_path = bulk_update.file.current_path
+      started_at = Time.zone.now
+
       File.open(csv_path) do |f|
         progress&.total = f.each_line.count(&:present?) - 1 # ignore the header
 
         ::CSV.table(f, header_converters: header_converter).each do |row|
           process_row(exhibit, row)
           progress&.increment
+          ActionCable.server.broadcast 'progress_channel',
```

Comment:
Should this be a different channel from the one the indexing uses?

Reply:
They use the same JS file. I would need to update a bunch of JS.

Reply:
To be clear, I'm not saying that it should be. I'm just asking whether you considered it and why you decided that one channel is the best approach.

Reply:
I considered it. Seemed like I would just be adding another listener for a different channel that would do the exact same thing. 

https://github.com/projectblacklight/spotlight/pull/3182/files#diff-a9fcc060ea558da08359b8855bc3feb142bf3f85f9ab5039fa55e82739a310c6R12-R18